tests/runtimes/simple-keyvalue/Cargo.toml

```diff
@@ -4,6 +4,16 @@ version = "0.3.0-alpha"
 authors = ["Oasis Labs Inc. <info@oasislabs.com>"]
 edition = "2018"
 
+[[bin]]
+name = "simple-keyvalue"
```

Comment:
Nice! Can you similarly update the `simple-keymanager-upgrade` so we don't need two crates? (similar `println` change should be fine there as well)

Reply:
Will do. The `println`s aren't actually needed, from what I've seen (the ones in here are just for debugging the PR).